src/modules/billing/services/two-part-tariff-service/prepare-returns.js

```diff
@@ -80,6 +80,12 @@ const checkDueReturnsErrors = returns => {
   }
 };
 
+const checkReturnsAndLines = returns => {
+  if (returns.length === 0) return true;
+  const returnsWithLines = returns.map(ret => !!ret.lines);
+  return returnsWithLines.includes(false);
+};
+
```

Comment:
I think this function name could be improved to indicate what is being checked for

It might also be able to be shortened to 

```
return returns.length === 0
  ? true
  : returns.some(ret => !ret.lines)
```

Reply:
Thank Mark :) I have made the suggested changes and have added a test for this scenario as I realised that I didn't add a test for a the missing lines scenario